setup.py

```diff
@@ -60,7 +56,7 @@ def run_tests(self):
 PACKAGES = [NAME.replace('-', '_'), 'django_dummysign']
 REQUIREMENTS = [
     # BEGIN requirements
-    'Django>=1.8',
+    'Django>=1.11,<2.0' if IS_PYTHON2 else 'Django>=1.11',
```

Comment:
Same remark than for another of our libraries, I think that `>=1.11`is enough. It's Django's job to say to pip "Don't install the >=2.0 versions with Python 2".

Reply:
makes sense!